projects/kit/components/input-tag/input-tag.component.ts

```diff
@@ -269,6 +269,10 @@ export class TuiInputTagComponent
         );
     }
 
+    get hasNativePlaceholder(): boolean {
+        return !!this.nativeFocusableElement?.placeholder;
```

Comment:
There's a `placeholder` input, no need to reach to actual element with a getter

Reply:
fixed